components/CheckDocumentation/vacuum/MxidWraparound.tsx

```diff
@@ -0,0 +1,116 @@
+        exceeds the threshold, specified in{" "}
+        <code>autovacuum_multixact_freeze_max_age</code>.
+      </p>
+      <p>
+        In order to avoid wraparound failure, old multixact IDs must be freezed
+        by VACUUMs and this is not avoidable. The more old multixact IDs need to
+        be freezed, the more expensive VACUUM costs, potentially causes the
+        overall performance degradation or simply takes really long time to
+        finish. The anti-wraparound autovacuum holds{" "}
+        <code>SHARE UPDATE EXCLUSIVE</code> lock, which can block DDL
+        statements.
+      </p>
+      <p>
+        If multixact ID space utilization reaches to 99.85% (3M multixact left),
+        the system will shut down and refuse to start any new multixact. This is
+        to prevent any data corruption from happening by running out multixact
+        ID. Resolving this requires manual intervention.
+      </p>
+      <p>
+        With Postgres 14+, there is a special type of failsafe VACUUM which
+        takes extraordinary measures to avoid the shutdown. While this is very
+        useful, you still want to avoid this as a failsafe autovacuum ignores
+        resource utilization constraints and can have significant performance
+        impact.
+      </p>
+      <h4>Common Causes</h4>
+      <ul>
+        <li>
+          <h5>Ineffectual autovacuum settings</h5>
+          <p>
+            Autovacuums, especially anti-wraparound autovacuums, are meant to
+            freeze old multixact IDs and keep utilization low in order to
+            prevent wraparound from happening. When autovacuum is turned off, or
+            autovacuum settings are not well suited to actual database usage, it
+            is possible that autovacuum is not able to keep up with freezing old
+            multixact IDs and causes utilization to grow. Make sure that
+            autovacuum is turned on, and revisit the configuration settings to
+            ensure that autovacuum will keep multixact ID space utilization
+            under control. You can check out the configuration settings related
+            to freezing in <Link to={serverVacuumFreezingUrl}>Freezing</Link>{" "}
+            page in VACUUM Advisor.
+          </p>
+        </li>
+        <li>
+          <h5>Blocked autovacuums</h5>
+          <p>
+            Even if autovacuum settings are adequate, it is possible that
+            autovacuums themselves are blocked by something, hence freezing (to
+            reduce multixact ID space utilization) is not able to keep up. The
+            typical blocker is a long running transaction, or a transaction
+            holding some locks that could cancel autovacuums. It is important to
+            make sure that there is no such transactions so that VACUUMs can
+            make freezing progress. Check for any long-running transactions or
+            its lock state on the <Link to={backendsUrl}>Connections</Link>{" "}
```

Comment:
```suggestion
            make freezing progress. Check for any long-running transactions and
            their lock state on the <Link to={backendsUrl}>Connections</Link>{" "}
```

If there is a lock state, there's a long-running transaction, right? I guess you could have a transaction without a lock state, but since you can't have it the other way around, I think "and" is more natural unless we want to reword this more elaborately.